web/Services/LandlordService.cs

```diff
@@ -100,4 +109,26 @@ public async Task ApproveLandlord(int landlordId,  BricksAndHeartsUser user)
         landlord.ApprovalAdminId = user.Id;
         await _dbContext.SaveChangesAsync();
     }
+    
+    public async Task<ILandlordService.LandlordRegistrationResult> EditLandlordDetails(LandlordProfileModel createModel, int? selectedUserId)
+    {
+        var landlordToEdit = await _dbContext.Landlords.SingleAsync(l => l.Id == selectedUserId);
+        landlordToEdit.Title = createModel.Title;
+        landlordToEdit.FirstName = createModel.FirstName;
+        landlordToEdit.LastName = createModel.LastName;
+        landlordToEdit.CompanyName = createModel.CompanyName;
+        landlordToEdit.Email = createModel.Email;
+        landlordToEdit.Phone = createModel.Phone;
+
+        _dbContext.Update(landlordToEdit);
+        await _dbContext.SaveChangesAsync();
+        return ILandlordService.LandlordRegistrationResult.Success;
+    }
+
+    public bool CheckForDuplicateEmail(LandlordProfileModel createModel, int? selectedUserId)
+    {
+        var editedLandlord = _dbContext.Landlords.Single(l => l.Id == selectedUserId);
+        return _dbContext.Landlords.SingleOrDefault(l => l.Email == createModel.Email) != null 
+               && editedLandlord.Email != createModel.Email;
+    }

```

Comment:
There's some odd names here that make this a bit confusing.

* `createModel` isn't really a create model here, it's an `editModel`, I guess?
* `selectedUserId` is a landlord id, not a user id. I also don't think we need it at all now that `LandlordProfileModel` has the id in itself
* The name `CheckForDuplicateEmail` doesn't tell us what the return value means, I think it would be clearer to name it something like `EditWouldCreateDuplicateEmail`